exporter/collector/integrationtest/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/collector/internal/integrationtest
 
-go 1.17
+go 1.16
```

Comment:
can we leave this at 1.17?

Reply:
Sure.   It's kinda wierd how half the project is 1.16 and the rest 1.17.  Should we have the whole thing be 1.17 or is this a backward compatible thing?

Reply:
We want the whole thing on 1.17, which I thought I had already done (but must've missed one).  Feel free to include that in this PR.

Reply:
Fixed